Source/ui_ios/EmulatorViewController.mm

```diff
@@ -63,6 +65,18 @@ - (void)viewDidLoad
 	[self.view addSubview:self.iCadeReader];
 	self.iCadeReader.delegate = self;
 	self.iCadeReader.active = YES;
+    
+    self.saveStateController = [[SaveStateController alloc] init];
```

Comment:
Is there a way to create this through the interface builder? I'm trying to avoid adding UI elements programatically, I find it easier to manipulate them in the editor.

Reply:
Yes it can be done through a storyboard using interface builder as well. I prefer creating the UI programmatically because:
- All the UI definition is in one place, no need to make connections with IBOutlets so less room for confusion
- Interface Builder can get really slow
- Manipulating autolayout constraints in Interface Builder is kind of a pain

I have no problem doing this with a storyboard though, if that's what you prefer.